test/blackbox-tests/test-cases/odoc-unique-mlds/run.t

```diff
@@ -8,6 +8,8 @@ Duplicate mld's in the same scope
           odoc _doc/_odoc/lib/root.lib2/root_lib2.odoc
           odoc _doc/_html/root/Root_lib2/.dune-keep,_doc/_html/root/Root_lib2/index.html
           odoc _doc/_odoc/pkg/root/page-index.odoc
+  File "../../../_mlds/root/index.mld", line 3, characters 0-21:
+  '{2': heading level should be lower than top heading level '2'.
```

Comment:
Where is the code that generates this `.mld` file?

Reply:
You probably answered your own question, but it's here:

https://github.com/rgrinberg/jbuilder/blob/fix-odoc-errors/src/odoc.ml#L418-L442